dpx/src/dpx_pdfobj.rs

```diff
@@ -123,9 +129,49 @@ impl pdf_obj {
     pub fn is_array(&self) -> bool {
         PdfObjType::from(self.typ) == PdfObjType::ARRAY
     }
+    pub fn get_array_mut(&mut self) -> &mut PdfArray {
```

Comment:
maybe better `as_array_mut` like in `lopdf`?

Reply:
Agree